src/Bicep.Cli/Services/TemplateEvaluatorService.cs

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using System;
+using Newtonsoft.Json.Linq;
+using Azure.Deployments.Core.Configuration;
+using Azure.Deployments.Core.Definitions.Schema;
+using Azure.Deployments.Templates.Engines;
+using Azure.Deployments.Expression.Engines;
+using System.Linq;
+using Azure.Deployments.Expression.Expressions;
+using Azure.Deployments.Core.ErrorResponses;
+using Microsoft.WindowsAzure.ResourceStack.Common.Extensions;
+using Microsoft.WindowsAzure.ResourceStack.Common.Collections;
+using System.Collections.Immutable;
+using Bicep.Core;
+
+namespace Bicep.Cli.Services
+{
+    public class TemplateEvaluatorService
+    {
+        const string TestTenantId = "d4c73686-f7cd-458e-b377-67adcd46b624";
```

Comment:
This is me being nit-picking, but I'd suggest using the name `Dummy*` or `Fake*` instead of `Test*` for reprensenting fake data. Also, it might be better if we can replace the GUIDs with empty GUIDs to make it obvious that they are not real.

Reply:
A related question - do we have any plan to allow users to mock these deployment environment data?